docs/src/contributing.md

```diff
@@ -122,6 +122,9 @@ Then, if all goes according to plan, it gets merged... **Thanks for the contribu
 
 ## Note on Adding Dependencies
 
+As a rule, we try to avoid having too many dependencies.
+Therefore, we request that if you have a PR that adds a new dependency, please have opened an issue previously and gotten express permission from `Javis.jl` maintainers to work on this.
```

Comment:
I think they should open an issue even if they don't add a dependency. Otherwise someone else might works on it. They can open the issue and work on the change directly. No permission needed for dependencies it would probably be good if we can say them beforehand that we don't want this. I would rephrase the word "permission" a bit :smile:

Reply:
Reworked this!